config/index.js

```diff
@@ -75,7 +75,8 @@ class RainbowConfig {
     const res = this.maybeGetParam(name)
     if (res === null) {
       console.error(`Parameter ${name} must be specified.`)
-      process.exit(1)
+      throw new Error('error')
```

Comment:
would be better to throw new Error(`Parameter ${name} must be specified.`) and avoid console.error

Reply:
Will be fixed in https://github.com/near/rainbow-bridge-lib/pull/13